Code/GraphMol/SubstructLibrary/Wrap/SubstructLibraryWrap.cpp

```diff
@@ -44,13 +44,13 @@ const char *MolHolderBaseDoc =
     "The API is quite simple: \n"
     "  AddMol(mol) -> adds a molecule to the molecule holder, returns index of "
     "molecule\n"
-    "  GetMol(idx) -> return the molecule at index idx\n";
+    "  GetMol(idx,sanitize=True) -> return the molecule at index idx\n";
```

Comment:
why is the `sanitize=True` here in the docs?

Reply:
It shouldn't be, that was from a previous iteration.